mcs/class/System/winaot_System.dll.sources

```diff
@@ -1 +1 @@
-#include common.sources
+#include mobile_System.dll.sources
```

Comment:
This looks wrong as it adds FileSystemWatcher for mobile profiles instead of default one